Sources/ClientRuntime/Networking/Http/Middlewares/URLPathMiddleware.swift

```diff
@@ -5,13 +5,15 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-public struct URLPathMiddleware<OperationStackInput: URLPathProvider, OperationStackOutput>: Middleware {
+public struct URLPathMiddleware<OperationStackInput, OperationStackOutput>: Middleware {
```

Comment:
This middleware no longer requires the input to conform to a protocol, and takes a closure at initialization to get the needed URL path instead.